src/main/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotStore.scala

```diff
@@ -82,6 +85,7 @@ class CassandraSnapshotStore(cfg: Config) extends SnapshotStore with CassandraSt
       // try initialize early, to be prepared for first real request
       preparedWriteSnapshot
       preparedDeleteSnapshot
+      preparedDeleteSnapshots
```

Comment:
We only want to do this one if we're on 3.0 upwards, as the prepare would fail.

We don't have any integration tests against old version of C* to catch this, unfortunately.

Reply:
Done.